app/assets/javascripts/details/app.js

```diff
@@ -1,3 +1,19 @@
 $(document).ready(function(){
-  // Do something here
+
+  //Attachment functionality
+  //Auto submits form
+  //Shows loading file
+  $(document).on('change','.attachment-file-input',function(){
```

Comment:
Heres how the other JS tests are being written: https://github.com/18F/C2/blob/new-details-page/edit-mode/spec/javascripts/state/edit_mode_spec.js.coffee